testing/marketplace/deployer/helm_tiller_onbuild/helm-dependency-build/chart/chartmuseum-marketplace/requirements.yaml

```diff
@@ -0,0 +1,7 @@
+dependencies:
+- name: marketplace-integration
+  version: 0.0.1
```

Comment:
Just a note that this doesn't have a be a fixed version. This is where `requirements.yaml` and `requirements.yaml.lock` differ. The lock file points to a specific version, while the yaml file specifies the constraints of what versions are accepted. `helm dependency update` pulls the latest versions that satisfy the constraints, while `helm dependency build` downloads the tarballs of those specific versions.

See [here](https://github.com/Masterminds/semver#checking-version-constraints) for more info on how version constraints are specified.

Probably for this, we want to specify `0.0.x` versions?

Reply:
Done!